packages/react/src/components/Popup/Popup.tsx

```diff
@@ -596,6 +596,7 @@ export default class Popup extends AutoControlledComponent<PopupProps, PopupStat
       this.trySetOpen(false, e)
       onClose && onClose()
       e.stopPropagation()
+      e.preventDefault()
```

Comment:
For me this change is unclear. Can we ship it as small PR with meaningful description?